src/test/shared/featureController.test.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { FeatureController } from '../../shared/featureController'
+import { TestSettingsConfiguration } from '../utilities/testSettingsConfiguration'
+
+describe('FeatureController', async () => {
+
+    const testConfig = new TestSettingsConfiguration()
```

Comment:
I see testConfig mutations in some of the tests. I think all of your initializations should occur `beforeEach` test, to ensure you don't introduce side-effects into other tests.

Reply:
✔️